components/kyma-environment-broker/common/orchestration/dto.go

```diff
@@ -8,13 +8,18 @@ import (
 
 // Parameters hold the attributes of orchestration create (upgrade) requests.
 type Parameters struct {
-	Targets  TargetSpec   `json:"targets"`
-	Strategy StrategySpec `json:"strategy,omitempty"`
-	DryRun   bool         `json:"dryRun,omitempty"`
+	Targets    TargetSpec           `json:"targets"`
+	Strategy   StrategySpec         `json:"strategy,omitempty"`
+	DryRun     bool                 `json:"dryRun,omitempty"`
+	Kubernetes KubernetesParameters `json:""`
 	// upgrade kyma specific parameters
 	Kyma KymaParameters `json:""`
```

Comment:
I think because there will be quite a few kyma & kubernetes parameters, have the block set in the JSON:

**BREAKING CHANGE, so CLI must be recompiled with the new common/orchestration package** (which you will anyway do as CLI changes are coming)

```suggestion
	Kubernetes *KubernetesParameters `json:"kubernetes,omitempty"`
	// upgrade kyma specific parameters
	Kyma *KymaParameters `json:"kyma,omitempty"`
```